src/azure-cli/azure/cli/command_modules/appconfig/_kv_helpers.py

```diff
@@ -539,18 +605,12 @@ def __export_keyvalues(fetched_items, format_, separator, prefix=None):
         raise CLIError("Fail to export key-values." + str(exception))
 
 
-def __export_features(retrieved_features, format_):
+def __export_features(retrieved_features, naming_convention):
+    feature_reserved_keywords = FeatureManagementReservedKeywords(naming_convention)
     exported_dict = {}
+    exported_dict[feature_reserved_keywords.featuremanagement] = {}
```

Comment:
nit: exported_dict = {feature_reserved_keywords.featuremanagement:{}}

Reply:
done

---
In reply to: [361023316](https://github.com/Azure/azure-cli/pull/11637#discussion_r361023316) [](ancestors = 361023316)